### ISSUE TITLE
Links to lib/**/*.pod go to metacpan rather than to the HTML

### ISSUE THREAD

**Issue by cxw42** [mgruner/p5-pod-projectdocs#10]
Thank you for this module, which is very useful!  I found an apparent bug in the handling of `.pod` files.

Steps to reproduce:
- `git clone https://github.com/cxw42/pod-projdocs-issue-example.git`.  This is a dist called "Acme-PPD" (Pod Project Docs).
- `cd pod-projdocs-issue-example`
- `./projdocs` (which does `pod2projdocs -o html -l lib`)
- open `./html/index.html` in a browser.  You will see a POD page and a .pm: 
![image](https://user-images.githubusercontent.com/10515894/104857739-962b2900-58e8-11eb-98da-1d370d8e93dc.png)
- click on `Acme/Manual`.  You will see a link to the Acme::PPD Acme/PPD.pm documentation: 
![image](https://user-images.githubusercontent.com/10515894/104857797-e4d8c300-58e8-11eb-8641-714f90df1a10.png).  Click on that link.  
- In the Acme::PPD documentation, you will see a link to `Acme::PPD::Manual`.  Hover over it to see its URL.

Expected:
- The link to `Acme::PPD::Manual` points to `file:///.../html/Acme/Manual.pod.html`

Observed:
- The link to `Acme::PPD::Manual` points to `http://metacpan.org/module/Acme::PPD::Manual`: 
![image](https://user-images.githubusercontent.com/10515894/104857874-6892af80-58e9-11eb-996c-c3b7a54a6da4.png)

It appears that `.pod` files don't get considered as local files when P::PD is making links.  Would you be willing to change this so that I can link to local `.pod` files?  Thank you for considering this request!

**Comment by mgruner** [comment]
Hi @cxw42. Thanks so much for the excellent report and pull request with tests. I merged it with a slight change: file name handling for `.pod` files is now in a more proper place in `File.pm`. 
Would you be so kind to review this and let me know if this works for you as well? Then I can release it to CPAN. 

Thanks again, and best regards,
Martin

**Comment by cxw42** [comment]
Martin, thanks very much for the quick response!  I'll try it out this week :) .

Chris

**Comment by cxw42** [comment]
@mgruner
- It does appear to render the links to the local manuals correctly!
- However, I get a JS error that prevents the main page from rendering:
    ```
    Uncaught TypeError: Cannot read property 'length' of undefined
        at get_rows_html (index.html:29)
        at render (index.html:14)
        at onload (index.html:58)
    ```
  The relevant line is:
    ```
    for (var i = 0; i < manager.records.length; i++) {
    ```
  As far as I can tell, this should be `manager.entries.length` in keeping with the changes to `Pod::ProjectDocs::gen()`.

**Comment by mgruner** [comment]
@cxw42 ah, right. Shame on me for relying on the automated tests only (which don't cover the JS integration)! Thanks for spotting this. I fixed this and released a new v0.53.

**Comment by cxw42** [comment]
Looks great --- much appreciated!
